src/handlers/user-start-stop.ts

```diff
@@ -10,8 +10,8 @@ export async function userStartStop(context: Context): Promise<{ output: string
   const { isEnabled } = config;
```

Comment:
This enabled stuff should be removed.

Reply:
I guess this was to mimic the v1 that actually had a field to disable the start stop command, which I don't think is needed for v2.

Reply:
I disagree. How can we disable it on specific repos with this removed without adding a config item to disable it for repos?

Reply:
Currently, the repo configuration will override the organization repo. There is an issue on that topic [here](https://github.com/ubiquity/ubiquibot-kernel/issues/54).

I would say that if in the target repository configuration, the plugin is not mentioned, it won't run (or be disabled if you will). If it is present, the target repo configuration will be used.

Reply:
> I would say that if in the target repository configuration, the plugin is not mentioned, it won't run (or be disabled if you will). If it is present, the target repo configuration will be used.

So it would be active globally via the org and if there is no config in the repo or there is but no mention of the plugin then it's disabled in that repo? I think I must misunderstand as that would mean it would be mandatory to define a repo config with this plugin wherever it should be "enabled"?

Reply:
No. If it is enabled organization level, it will be enabled in every repo. I a repo defines a configuration with `command-start-stop` in it, it will override totally the organization configuration. If a repo defines a configuration without `command-start-stop` then it will never run on that repo.

Reply:
Right okay I'm following so far. 

So the pattern we've been using like `config?.item ? action() : null` is that still relevant for determining if a plugin should run? Most recently it happened in [xpCheck](https://github.com/ubiquibot/command-start-stop/pull/17/files#diff-1181dad3ddb1451203d2941f1b46bf3c3ea60bcc8e0da7093513931d0d11e518) but that's for a sub-feature of the plugin I guess, so would that pattern apply in similar scenarios for enabling/disabling specific features?

Reply:
Yes, implemented [here for example](https://github.com/ubiquibot/conversation-rewards/pull/75#issuecomment-2253656887).

Reply:
> No. If it is enabled organization level, it will be enabled in every repo. I a repo defines a configuration with `command-start-stop` in it, it will override totally the organization configuration. If a repo defines a configuration without `command-start-stop` then it will never run on that repo.

Can you demonstrate how the config would look if you wanted to disable it on a repo?

Reply:
```yml
plugins:
    # any plugin you want except this one
```

Reply:
This doesn't seem right. Configuration merging should be at the resolution of the plugin id level. And I would have expected that in order to disable this we would have something like 
```yml
plugins:
   issues_comment.created:
      - id: ubiquibot/command-start-stop
        with: 
```

Reply:
> if you cannot see the configuration file.

The bot has access to private repositories. Using the bot authentication, you should be able to access the organization config.